extensions/listeners.py

```
@@ -85,8 +88,30 @@ async def on_interaction(self, interaction: discord.Interaction) -> None:
             elif custom_id.startswith("ticket_close"):
                 await closeTicketListener(interaction)
                 return
+        except discord.Forbidden:
+            await self._send_error(interaction, "I don't have permission to do that.")
+        except discord.NotFound:
+            await self._send_error(interaction, "This interaction has expired.")
+        except discord.HTTPException as e:
+            await self._send_error(interaction, f"Discord API error (code {e.status}).")
+        except Exception as e:
```

Flake8 (7.3.0)

[error] local variable 'e' is assigned to but never used

(F841)